partners.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <link rel="stylesheet" href="/index.css">
+  <title>About-page</title>
+</head>
+<body>
+  <header class="header-container" >
+    <div class="first-section-nav">
+      <ul class="section-nav-list">
+        <li ><img class="section-nav-item" src='/assets/facebook.png' alt="logo"></li>
+        <li ><img class="section-nav-item" src='/assets/twitter.png' alt="logo"></li>
+        <li>French</li>
+        <li>English</li>
+        <li>Log out</li>
+      </ul>
+    </div>
+    <div class="header">
+      <button class="moile-menu-hamburger">
+        <span class="hamburger"></span>
+      </button>
+      <h2 class="logo-uncc"> <a href="/index.html">UNCC</a></h2>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a href="/partners.html">About</a></li>
+          <li class="nav-item"><a href="#" class="black">Program</a></li>
+          <li class="nav-item"><a href="#" class="black">Join</a></li>
+          <li class="nav-item"><a href="#partners" class="black">Partners</a></li>
+          <li class="nav-item"><a href="#" class="black">News</a></li>
+          <li class="nav-item campaign"><a href="#"><p>AT Campaign</p></a></li>
+        </ul>
+      </nav>
+    </div>
+    
+  </header>
+  <main>
+    <section class="section-one">
+      <h1 class="section-one-title">UNITING THE WORLD
+        TO TACKLE CLIMATE CHANGE.
+      </h1>
+      <p class="presentation-description">
+        The COP26 summit brought parties together to accelerate 
+        action towards the goals of the Paris Agreement and the 
+        UN Framework Convention on Climate Change.
+        Learn about how the negotiations at COP26 went and the
+         outcomes achieved in the documents below.
+      </p>
+      <p class="contact-glascow">Please contact us per Email for any further 
+        questions about The Glascow conference. <a href="#">contact@atr.org</a></p>
+    </section>
```

Comment:
- [ ] The text for this section should be aligned to the center and not the left side. Kindly implement this. 👍 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/157769720-6b50cc26-bf85-4f7e-8595-0834ccfa44ba.png)

### Behance Design

![image](https://user-images.githubusercontent.com/77106962/157769908-f92acf67-8690-4e6b-a8cb-b613be70779b.png)